src/models/model.cpp

```diff
@@ -204,6 +204,12 @@ std::string Tokenizer::Decode(std::span<const int32_t> tokens) const {
   return string;
 }
 
+std::string Tokenizer::ApplyChatTemplate(const char* template_str, const char* messages, bool add_generation_prompt) const {
+  const char* output;
```

Comment:
```suggestion
  const char* output{};
```
Zero init for safety.